ipywidgets/widgets/widget_selection.py

```diff
@@ -23,6 +23,40 @@
                        Tuple, List, Union, observe, validate)
 from ipython_genutils.py3compat import unicode_type
 
+_selection_parameters_docs = """
+    options: dict or list
+        The options for the dropdown. This can either be a
```

Comment:
Can we 'demote' the dictionary representation and put in a note about order? Also, add an example where the values are not just strings?

> This can either be a list of values, e.g. ``['Galileo', 'Brahe', 'Hubble']`` or ``[0, 1, 2]``, a list of (label, value) pairs, e.g. ``[('Galileo', 0), ('Brahe', 1), ('Hubble', 2)]``, or a dictionary mapping the labels to the values, e.g. ``{'Galileo': 0, 'Brahe': 1, 'Hubble': 2}`` (in the case of a dictionary, the order of options is not guaranteed).